.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
```

Comment:
Don't you need a setup.cfg with these rules? Otherwise you'll get black not making your lines too long and flake8 complaining about it.
```
[isort]
profile = black

[flake8]
max-line-length = 88
extend-ignore = E203, W503
```

Reply:
yes, I was wondering why flake8 was enforced on top of black's 88 max-line-length in VS code. I was missing a `setup.cfg` file. Thank you @ovesh !